parquet/src/main/scala/magnolify/parquet/ParquetType.scala

```diff
@@ -58,6 +52,17 @@ sealed trait ParquetType[T] extends Serializable {
   import ParquetType._
 
   def schema: MessageType
+  def avroSchema: AvroSchema = {
+    val messageSchema = Schema.message(schema)
+    val avroSchema = new AvroSchemaConverter().convert(messageSchema)
+    SchemaUtil.deepCopy(
+      avroSchema,
+      typeAnnotation,
+      path => fieldAnnotations.get(path)
+    )
+  }
+  def fieldAnnotations: Map[String, String]
+  def typeAnnotation: Option[String]
```

Comment:
This should not be exposed at the  `ParquetType` level

Reply:
Done